### PR TITLE
Lercerss/front end/theme

### PR DESCRIPTION
**Pull request by niiita**
Modifications for unifying front end for all existing pages.
Solves issues #55 #51 #56.

### CODE REVIEW

File: SA3/qa_web/templates/qa_web/answerspage.html [hunk 1]
```diff
@@ -42,71 +38,80 @@
...
+            {% if user.is_authenticated and user == currentQuestion.owner and not bestAnswer %} 
+                <input type = "submit" name="select_{{ answer.id }}" value="Select as Best Answer"></input>
+            {% endif %}
+            <h6>Answer by {{ answer.owner }} on {{ answer.creation_date }}</h6>
+        </form>
+        <a id='upvote_{{ answer.id }}_answer'><button>Upvote</button></a>
+        <span id='score_{{ answer.id }}_answer'>{{ answer.score }}</span>
+        <a id='downvote_{{ answer.id }}_answer'><button>Downvote</button></a>
+        <hr>
+    {% empty %}
```
Comment (jetcanfly) [inline SA3/qa_web/templates/qa_web/answerspage.html]: Good practice of tag.

File: SA3/qa_web/templates/qa_web/question_display_page.html [hunk 2]
```diff
@@ -1,20 +1,59 @@
...
-        <li role="presentation"
-            {% if active_tab == 'latest' %}class="active"{% endif %}>
+    <ul class="nav nav-tabs">
+
+        <li class="active" {% if active_tab == 'latest' %}class="active"{% endif %}>
+            <a href="#latest" data-toggle="tab">
+                <span class="fas fa-clock"></span>
+                    Latest</a>
+        </li>
+<li {% if active_tab == 'hottest' %}class="active"{% endif %}>
```
Comment (jetcanfly) [inline SA3/qa_web/templates/qa_web/question_display_page.html]: I've implemented a tab for questions that are not answered yet. I could add these tabs' view logic in next sprint.

File: SA3/qa_web/templates/qa_web/includes/navbar.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        <a class="navbar-brand" href="#">Sign In to Ask, Answer &amp; Rate</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+            <ul class="navbar-nav ml-auto">
+                {% if user.is_authenticated %}
+                    <li class="nav-item">
+                        {# A link to the user profile should be added here #}
+                        <a class="nav-link" href="#">Welcome {{ user.username }}</a>
```
Comment (alvyn279) [inline SA3/qa_web/templates/qa_web/includes/navbar.html]: I like this welcome message, it helps to know if we are logged in or not.

File: SA3/qa_web/templates/qa_web/base.html [hunk 4]
```diff
@@ -7,36 +7,67 @@
...
+</div>
+
+
+    {% include "qa_web/includes/footer.html" %}
+
+
+
+
+    <!-- JQuery -->
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```
Comment (alvyn279) [inline SA3/qa_web/templates/qa_web/base.html]: I feel like the js can be loaded in the `<head>` tag so we could see all of our static files in them same place for convenience. But if this is a good practice and our features work then we can definitely keep it this way.

**Comment by jetcanfly** [comment]
LGTM

**Pull request merged**